### PR TITLE
Keysend Fix

### PR DESCRIPTION
**Pull request by myxmaster**
# Description

Fixed Keysed by fixing hexToBase64. Also refactored base64ToHex.

This pull request is categorized as a:

- [ ] New feature
- [x] Bug fix
- [ ] Code refactor
- [ ] Configuration change
- [ ] Locales update
- [ ] Quality assurance 
- [ ] Other

## Checklist
- [ ] I’ve run `yarn run tsc` and made sure my code compiles correctly
- [x] I’ve run `yarn run lint` and made sure my code didn’t contain any problematic patterns
- [x] I’ve run `yarn run prettier` and made sure my code is formatted correctly
- [x] I’ve run `yarn run test` and made sure all of the tests pass

## Testing

If you modified or added a utility file, did you add new unit tests?

- [ ] No, I’m a fool
- [ ] Yes
- [x] N/A

I have tested this PR on the following platforms (please specify OS version and phone model/VM):

- [x] Android
- [ ] iOS

I have tested this PR with the following types of nodes (please specify node version and API version where appropriate):

- [ ] Embedded LND
- [x] LND (REST)
- [ ] LND (Lightning Node Connect)
- [ ] Core Lightning (c-lightning-REST)
- [ ] LndHub
- [ ] [DEPRECATED] Core Lightning (Spark)
- [ ] [DEPRECATED] Eclair

### Locales
- [ ] I’ve added new locale text that requires translations
- [ ] I’m aware that new translations should be made on the Zeus [Transfix page](https://app.transifex.com/ZeusLN/zeus/) and not directly to this repo

### Third Party Dependencies and Packages

- [ ] Contributors will need to run `yarn` after this PR is merged in
- [ ] 3rd party dependencies have been modified:
    * verify that `package.json` and `yarn.lock` have been properly updated
    * verify that dependencies are installed for both iOS and Android platforms

### Other:

- [ ] Changes were made that require an update to the README
- [ ] Changes were made that require an update to onboarding

### CODE REVIEW

File: utils/Base64Utils.ts [hunk 1]
```diff
@@ -46,15 +23,8 @@ class Base64Utils {
...
-        const raw = this.decodeBase64ToString(base64String);
-        let result = '';
-        for (let i = 0; i < raw.length; i++) {
-            const hex = raw.charCodeAt(i).toString(16);
-            result += hex.length === 2 ? hex : '0' + hex;
-        }
-        return result;
-    };
+    base64ToHex = (base64String: string) =>
+        Buffer.from(base64String, 'base64').toString('hex');
```
Comment (kaloudis) [inline utils/Base64Utils.ts:27]: this function is a duplicate of `decodeBase64ToString`

**Review by kaloudis — CHANGES_REQUESTED** [review]

File: utils/Base64Utils.ts (hunk 1 above)
Comment (tkohlmeier) [reply utils/Base64Utils.ts:27]: The toString parameters differ. Here we convert to hex, in decodeBase64ToString to utf8.

**Review by kaloudis — APPROVED** [review]
tACK

**Comment by kaloudis** [comment]
Nice addition of tests.

**Pull request merged**